class-one-exchange-htb-exports.js

```diff
@@ -6,6 +6,11 @@
  *  window namespace here, but this is discouraged if it's not strictly needed.
  */
 
-shellInterface.ClassOneExchangeHtb = {
-    render: SpaceCamp.services.RenderService.renderDfpAd.bind(null, 'ClassOneExchangeHtb')
+shellInterface.C1XHtb = {
+    render: SpaceCamp.services.RenderService.renderDfpAd.bind(null, 'C1XHtb')
 };
+
+if (__directInterface.Layers.PartnersLayer.Partners.C1XHtb) {
+    shellInterface.C1XHtb = shellInterface.C1XHtb || {};
+    shellInterface.C1XHtb.adResponseCallbacks = __directInterface.Layers.PartnersLayer.Partners.C1XHtb.adResponseCallbacks;
+} // TO DO: check with IX about the callback naming. 
```

Comment:
Hello IX team,
Can you let me know should we put the exact callback name from our JSONP response returned by our bidder?Our response looks like this:
```
_c1xResponse([{
  bid: true,
  adId: 'gpt-36-right-center',
  cpm: 1.5,
  ad: '<div><a target="_new" href="http://c1exchange.com"><img src="http://c1x.s3.amazonaws.com/ads/house/banner300x250/c1x-ad2.jpg"></a></div>',
  width: 300,
  height: 250
}]);
```